jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/transform/node/incoming/TccGeneratorNode.kt

```diff
@@ -101,11 +125,39 @@ class TccGeneratorNode(
                 // TODO: Chrome does something more advanced, keeping older sequences to replay on packet reordering.
                 packetArrivalTimes.clear()
             }
-            if (windowStartSeq == -1 || tccSeqNum < windowStartSeq) {
```

Comment:
Generally this code path shouldn't ever be receiving packets without timestamps - the timestamp is just nullable in PacketInfo because we need to be able to handle locally-injected packets, but those shouldn't ever be being processed in this part of the pipeline.

If *no* packets had timestamps, `isTccReadyToSend` will never return true.  If a single packet lacks a timestamp for whatever reason, but we have other queued TCC packets, I think we'll just send the TCC feedback for the previous packet.  (The packet without a timestamp will get reported as lost if subsequent packets come in with a timestamp, which isn't great, but as I said I don't think this will ever occur in practice.)